airbyte-test-utils/src/main/java/io/airbyte/test/utils/AirbyteAcceptanceTestHelper.java

```diff
@@ -0,0 +1,721 @@
+    return createSource(
+        "acceptanceTestDb-" + UUID.randomUUID(),
+        workspaceId,
+        getPostgresSourceDefinitionId(),
+        getSourceDbConfig());
+  }
+
+  public SourceRead createSource(final String name, final UUID workspaceId, final UUID sourceDefId, final JsonNode sourceConfig)
+      throws ApiException {
+    final SourceRead source = apiClient.getSourceApi().createSource(new SourceCreate()
+        .name(name)
+        .sourceDefinitionId(sourceDefId)
+        .workspaceId(workspaceId)
+        .connectionConfiguration(sourceConfig));
+    sourceIds.add(source.getSourceId());
+    return source;
+  }
+
+  public UUID getPostgresSourceDefinitionId() throws ApiException {
+    return apiClient.getSourceDefinitionApi().listSourceDefinitions().getSourceDefinitions()
+        .stream()
+        .filter(sourceRead -> sourceRead.getName().equalsIgnoreCase("postgres"))
+        .findFirst()
+        .orElseThrow()
+        .getSourceDefinitionId();
+  }
+
+  public void clearSourceDbData() throws SQLException {
+    final Database database = getSourceDatabase();
+    final Set<SchemaTableNamePair> pairs = listAllTables(database);
+    for (final SchemaTableNamePair pair : pairs) {
+      database.query(context -> context.execute(String.format("DROP TABLE %s.%s", pair.schemaName, pair.tableName)));
+    }
+  }
+
+  public void clearDestinationDbData() throws SQLException {
+    final Database database = getDestinationDatabase();
+    final Set<SchemaTableNamePair> pairs = listAllTables(database);
+    for (final SchemaTableNamePair pair : pairs) {
+      database.query(context -> context.execute(String.format("DROP TABLE %s.%s CASCADE", pair.schemaName, pair.tableName)));
+    }
+  }
+
+  public void disableConnection(final UUID connectionId) throws ApiException {
+    final ConnectionRead connection = apiClient.getConnectionApi().getConnection(new ConnectionIdRequestBody().connectionId(connectionId));
+    final ConnectionUpdate connectionUpdate =
+        new ConnectionUpdate()
+            .prefix(connection.getPrefix())
+            .connectionId(connectionId)
+            .operationIds(connection.getOperationIds())
+            .status(ConnectionStatus.DEPRECATED)
+            .schedule(connection.getSchedule())
+            .syncCatalog(connection.getSyncCatalog());
+    apiClient.getConnectionApi().updateConnection(connectionUpdate);
+  }
+
+  public void deleteSource(final UUID sourceId) throws ApiException {
+    apiClient.getSourceApi().deleteSource(new SourceIdRequestBody().sourceId(sourceId));
+  }
+
+  public void deleteDestination(final UUID destinationId) throws ApiException {
+    apiClient.getDestinationApi().deleteDestination(new DestinationIdRequestBody().destinationId(destinationId));
+  }
+
+  public void deleteOperation(final UUID destinationId) throws ApiException {
+    apiClient.getOperationApi().deleteOperation(new OperationIdRequestBody().operationId(destinationId));
+  }
```

Comment:
it looks like some of these can be private instead of public - can we go through and double check these?

Reply:
done